commit 5a1257114b87084ed87c182f832c5697484c9475
Author: Scott Moser <smoser@brickies.net>
Date:   Tue Feb 6 23:02:59 2024 -0500

    fix compile error with gcc13 #1281 (#1340)
    
    Summary:
    Due to[ header dependency changes](https://gcc.gnu.org/gcc-13/porting_to.html#header-dep-changes) in GCC 13, we need to include the <cstdint> header.
    
    Pull Request resolved: https://github.com/facebookresearch/fastText/pull/1340
    
    Reviewed By: jmp84
    
    Differential Revision: D51602433
    
    Author: Cherilyn Buren <88433283+NiuBlibing@users.noreply.github.com>
    Pulled By: alexkosau
    
    fbshipit-source-id: cc9bffb276cb00f1db8ec97a36784c484ae4563a
    
    (cherry-picked from 6c2204ba66776b700095ff73e3e599a908ffd9c3)

diff --git a/src/args.cc b/src/args.cc
index 7ac8c2f..c7c0a6e 100644
--- a/src/args.cc
+++ b/src/args.cc
@@ -8,7 +8,8 @@
 
 #include "args.h"
 
-#include <stdlib.h>
+#include <cstdlib>
+#include <cstdint>
 
 #include <iostream>
 #include <stdexcept>
